shared/crypto/operations/decrypt/index.tsx

```diff
@@ -87,12 +88,14 @@ const Decrypt = (props: Props) => {
               outputType={props.outputType}
               textType="plain"
               operation={Constants.Operations.Decrypt}
+              outputMatchesInput={props.outputMatchesInput}
               onShowInFinder={props.onShowInFinder}
```

Comment:
This is a general comment for most of the component in this section, the children need to be connected more. We shouldn't be plumbing these props they can derive themselves